app/helpers/manipulation_helper.rb

```diff
@@ -0,0 +1,14 @@
+module ManipulationHelper
```

Comment:
I'm not sure this should be a top-level helper, because then it's global to the whole app...it should probably namespaced to the plugin.